contracts/MultiVesting.sol

```diff
@@ -875,19 +875,14 @@ contract MultiVesting is Ownable
 
         require(available >= _amount, "DON_T_HAVE_ENOUGH_PMA");
 
-        uint256 _rewardsPerMonth = 0;
-        if (_months > 0) {
-            _rewardsPerMonth = (_amount - _startAmount).div(_months);
-        }
-
-        Vesting memory v = Vesting({
+       Vesting memory v = Vesting({
             beneficiary: _beneficiary,
             startedAt : _startedAt,
             totalAmount : _amount,
             releasedAmount : 0,
-            startAmount:_startAmount,
```

Comment:
we actually do not need this. the getAvailable function take care of it

Reply:
But it will be good as in vesting map also we will getting right results .

Reply:
right